core/server/master/src/main/java/alluxio/master/block/meta/MasterWorkerInfo.java

```diff
@@ -189,9 +161,9 @@ public void removeBlock(long blockId) {
    * @param dirPath the lost storage path
    */
   public void addLostStorage(String tierAlias, String dirPath) {
-    List<String> paths = mLostStorage.getOrDefault(tierAlias, new ArrayList<>());
+    List<String> paths = mUsage.mLostStorage.getOrDefault(tierAlias, new ArrayList<>());
```

Comment:
is this method called with usage-lock held? If so can you update the method comment?